src/main/java/Task.java

```diff
@@ -0,0 +1,30 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    @Override
+    public String toString() {
+        return description;
+    }
```

Comment:
I like the use of polymorphism to output different kinds of `Task`s while making sure the output methods only have to deal with one class.